doc/source/_templates/sidebar-nav-bs.html

```diff
@@ -1,9 +1,9 @@
 <nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
   <div class="bd-toc-item active">
     {% if pagename.startswith("reference") %}
-    {{ generate_nav_html("sidebar", maxdepth=4, collapse=True, includehidden=True, titles_only=True) }}
+    {{ generate_nav_html("sidebar", maxdepth=1, collapse=True, includehidden=True, titles_only=True) }}
     {% else %}
-    {{ generate_nav_html("sidebar", maxdepth=4, collapse=True, includehidden=True, titles_only=True) }}
+    {{ generate_nav_html("sidebar", maxdepth=1, collapse=True, includehidden=True, titles_only=True) }}
```

Comment:
You might want a bigger maxdept for the page that are *not* in the reference section. For example, now "User Guide" -> "Statistics" doesn't show its subsections anymore

Reply:
I switched to 2. But IMO this would not be a problem as we have the toc on the right side.